src/views/ABIDecoder.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from "react";
+import {
+    Button,
+    Card,
+    CardHeader,
+    CardBody,
+    Row,
+    Col,
+} from "reactstrap";
+import '../../assets/css/black-dashboard-react.css';
+import InputField from "../components/inputField";
+import Caver from 'caver-js'
+import { networkLinks } from '../constants/klaytnNetwork'
+import Column from "../components/Column"
+let caver;
+
+class ABIDecoder extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            network: "mainnet",
+            result: "",
+        }
+    }
+
+    componentDidMount(){
+        caver = new Caver(new Caver.providers.HttpProvider(networkLinks[this.state.network]["rpc"]))
+    }
+
+    handleInputChange = (e) => {
+        const { name, value } = e.target;
+        this.setState({
+            [name]: value
+        })
+    }
+
+    decodeABI = async() => {
+        const { argumentTypes, encodedData } = this.state;
+        let typesArray;
+        let hexstring;
+        try{
+            typesArray = argumentTypes.split(' ');
+            hexstring = encodedData
+            console.log('typesArray', typesArray);
+            console.log('hexstring', hexstring);
+            const res = await caver.abi.decodeParameters(typesArray, hexstring)
+            console.log('res', res);
+            if (res){
+                let tempArr = []
+                for (let i = 0; i < res.__length__; i++){
+                    tempArr.push(res[i])
+                }
+                this.setState({ result: tempArr.toString() })
```

Comment:
How about using tempArr.join(", ")? I think space between results can make it easy to distinguish values.